.pre-commit-config.yaml

```diff
@@ -6,7 +6,7 @@ repos:
         exclude: conda/meta.yaml
  
   - repo: https://github.com/psf/black
-    rev: 22.6.0
+    rev: 23.1.0
```

Comment:
Thanks. Please also update `pre-commit-hooks` to `v4.4.0` and isort to `5.12.0`.

Reply:
Done in the commit: [3d707bf](https://github.com/lsst-ts/ts_m2/pull/40/commits/3d707bfc2083590a272eb294293b644d2636f631). Thanks!